skyportal/handlers/api/observation.py

```diff
@@ -412,11 +427,31 @@ def post(self):
                 'filter',
                 'exposure_time',
             ]
+        ) and not all(
```

Comment:
Can we replace these with `issubset`? Also the one above it while you're at it.

Reply:
yessir.